sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -44,7 +50,11 @@ import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, Project}
  *                 unless they are specified with full qualified table name with database prefix.
  */
 case class CreateViewCommand(
-    tableDesc: CatalogTable,
+    name: TableIdentifier,
+    userSpecifiedColumns: Seq[(String, Option[String])],
+    comment: Option[String],
+    properties: Map[String, String],
+    originalText: Option[String],
     child: LogicalPlan,
     allowExisting: Boolean,
     replace: Boolean,
```

Comment:
I'd like to do it in another PR, to move `SaveMode` into catalyst module and use it for CREATE TABLE/VIEW